Frontend/ui-customer/src/Screens/OnBoardingFlow/WelcomeScreen/ScreenData.purs

```diff
@@ -2,16 +2,55 @@ module Screens.OnBoardingFlow.WelcomeScreen.ScreenData where
 
 import Screens.Types (WelcomeScreenState)
 import Foreign.Object (empty)
+import Common.Styles.Colors as Color
+import Prelude (map)
+import Common.Types.App (YoutubeData, CarouselData)
 
 initData :: WelcomeScreenState
 initData = {
   data : {
-    carouselModel : [
-      {image : "carousel_1", title : "The fastest auto booking\napp is here!", description : "Our speedy booking process means\nyou get a ride quickly and easily."},
-      {image : "carousel_2", title : "No more\nsurge pricing!", description : "Experience fair and consistent fares,\neven during peak hours."},
-      {image : "carousel_4", title : "Inclusive and accessible for everyone!", description : "We strive to provide all our users an \n even & equal experience."},
-      {image : "carousel_3", title : "Be a part of the Open\nMobility Revolution!", description : "Our data and product roadmap are\ntransparent for all."}
-    ],
+    carouselModal : 
+      { gravity : 17
+      , carouselData : 
+          map(\item -> 
+            { imageConfig : { image : item.image , height : item.imageHeight , width : 200, bgColor : "#FFFAED", cornerRadius : 8.0 },
+              backgroundColor : "#FFFAED",
+              youtubeConfig : dummyYoutubeData,
+              gravity : item.gravity ,
+              contentType : "IMAGE" , 
+              titleConfig : {
+                text : item.title,
+                textSize : 22,
+                textColor : Color.black800,
+                gravity : "CENTER",
+                margin : { top : 10 , bottom : 0 , right : 20 , left : 20 }
+              }, 
+              descriptionConfig : {
+                text : item.description, 
+                textSize : 16,
+                textColor : Color.black700,
+                gravity : "CENTER",
+                margin : { top : 6 , bottom : 0 , right : 20 , left : 20 }
+              }
+            })
+              [ {image : "carousel_1", imageHeight : 260, title : "The fastest auto booking\napp is here!", description : "Our speedy booking process means <br> you get a ride quickly and easily.", gravity : 17},
```

Comment:
Is this data used ? 
if yes then Don't we need translations for these ?
If no then remove it

Reply:
yes it is required @Rohit4997 . This is being sent as an array to the map.